.github/workflows/test.yml

```diff
@@ -2,11 +2,20 @@ on: push
 name: Test installation
 
 jobs:
+  jest-tests:
+    name: "Run Jest tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
`[non-blocking]`
Note for future PR: latest version is `v4`. We might want to consider updating to it at some point.

Reply:
not big deal, updated to v4 in this PR